_includes/macros/mobile-carousel.html

```diff
@@ -0,0 +1,50 @@
+
+{# ==========================================================================
+
+   mobile_carousel.render()
+
+   ==========================================================================
+
+   Description:
+
+   Render HTML for a carousel of slides with an image, title, and description.
+   Argument is:
+
+   slides: An array of data to populate the carousel slides with.
+           The structure looks like:
+           {
+           'image_sizes_class': <A image sizing class for responsive sizes>,
+           'image_base_url':    <Base URL path for images>,
+           'data': [{
+              'image_filename': <Carousel slide image filename>,
+              'title':          <Carousel slide filename>,
+              'desc':           <Carousel slide description>
+              }, {
+              'image_filename': <Carousel slide image filename>,
+              'title':          <Carousel slide filename>,
+              'desc':           <Carousel slide description>
+              }]
+           }
+
+   ========================================================================== #}
+
+{% macro render( slides ) %}
```

Comment:
Removed parens in https://github.com/cfpb/cfgov-refresh/commit/103ef847f88472457936689fc4013cfaef2ab7f8 till we work this out.